tests/network_analysis_examples.py

```diff
@@ -0,0 +1,137 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
```

Comment:
Bruk percent-format når du lagrer Notebooks.  Trengs for at isort skal håndtere sortering av importer riktig i slike filer, med opsjonen` treat_comments_as_code = ["# %%"]`.